File: Java/22. Perfect number.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.println("Enter the Number to check Perfect:");
        int enteredNumber = userInput.nextInt();
        int add = 0;

        for(int i = 1; i<enteredNumber; i++) {
            if ( enteredNumber % i == 0 ) {
                add += i;
            }
        }
        if((add == enteredNumber) && (enteredNumber != 0)) {
            System.out.println(enteredNumber + " is a Perfect number");
        } else {
            System.out.println(enteredNumber + " is not a Perfect Number");
        }
        userInput.close();
    }
}

File: Python/01. Important Fundamentals/22. Perfect number.py
number = int(input("Enter the number to check Perfect:"))
add = 0

for i in range(1,number):
    if(number % i == 0):
        add += i

print(f"{number} is a Perfect Number" if((number!=0) and (number == add)) else f"{number} is not a Perfect number")